cmd/kube-controller-manager/app/options/persistentvolumebindercontroller.go

```diff
@@ -43,6 +43,7 @@ func (o *PersistentVolumeBinderControllerOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&o.VolumeConfiguration.EnableHostPathProvisioning, "enable-hostpath-provisioner", o.VolumeConfiguration.EnableHostPathProvisioning, "Enable HostPath PV provisioning when running without a cloud provider. This allows testing and development of provisioning features.  HostPath provisioning is not supported in any way, won't work in a multi-node cluster, and should not be used for anything other than testing or development.")
 	fs.BoolVar(&o.VolumeConfiguration.EnableDynamicProvisioning, "enable-dynamic-provisioning", o.VolumeConfiguration.EnableDynamicProvisioning, "Enable dynamic provisioning for environments that support it.")
 	fs.StringVar(&o.VolumeConfiguration.FlexVolumePluginDir, "flex-volume-plugin-dir", o.VolumeConfiguration.FlexVolumePluginDir, "Full path of the directory in which the flex volume plugin should search for additional third party volume plugins.")
+	fs.DurationVar(&o.VolumeOperationMaxBackoff.Duration, "volume-operation-max-backoff-time", o.VolumeOperationMaxBackoff.Duration, "The maximum backoff time of pv operation, defaults to 2minutes+2second.")
```

Comment:
It still feels iffy that we are defining an option on persistentvolumebindercontroller which applies to all the volume related controllers.I realize that perhaps there is no other "common" place to put this option but it still does not feel ideal.

Reply:
Yes，we can not found a common place. Do you I have some good ideas.
@gnufied @jingxu97